pkg/pquerier/chunkIterator_test.go

```diff
@@ -0,0 +1,111 @@
+// +build integration
+
+package pquerier_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"github.com/v3io/v3io-tsdb/pkg/config"
+	"github.com/v3io/v3io-tsdb/pkg/pquerier"
+	"github.com/v3io/v3io-tsdb/pkg/tsdb"
+	"github.com/v3io/v3io-tsdb/pkg/tsdb/tsdbtest"
+	"github.com/v3io/v3io-tsdb/pkg/utils"
+)
+
+type testRawChunkIterSuite struct {
+	suite.Suite
+	v3ioConfig     *config.V3ioConfig
+	suiteTimestamp int64
+}
+
+func (suite *testRawChunkIterSuite) SetupSuite() {
+	v3ioConfig, err := tsdbtest.LoadV3ioConfig()
+	if err != nil {
+		suite.T().Fatalf("unable to load configuration. Error: %v", err)
+	}
+
+	suite.v3ioConfig = v3ioConfig
+	suite.suiteTimestamp = time.Now().Unix()
```

Comment:
Can we use a constant time?

Reply:
This is for tsdb table naming format.
But i did change the actual `baseTime` i use in the test to be constant

Reply:
Ok, using a changing location makes sense of course, so tests are reliably repeatable.